Cargo.toml

```diff
@@ -42,7 +42,7 @@ bindgen = { version = "0.71.1", default-features = false, features = [] }
 bolero = { version = "0.13.0", default-features = false, features = [] }
 bytes = { version = "1.10.1", default-features = false, features = [] }
 # std feature currently globally required for clap
-clap = { version = "4.5.32", default-features = false, features = ["std"] }
+clap = { version = "4.5.32", default-features = true, features = [] }
```

Comment:
Do you need to set `default-features = true` here if you already specify the required features in `dataplane/Cargo.toml`?

Reply:
Apparently yes. I tried setting the features only in the dataplane crate and it did not work.

Reply:
Ok thanks